content/en/docs/concepts/instrumentation/libraries.md

```diff
@@ -67,6 +93,27 @@ still useful in others.
 The rest of this document gives guidance on what and how to instrument if you
 decide to do it.
 
+## Personally Identifiable Information (PII)
+
+In some semantic conventions, the data collected as a span attribute could
+contain PII (Personally Identifiable Information). As a general guideline, do
+**not** collect this data by default. A few other options for handling this data
+are:
+
+- Don't collect it at all
+- Provide opt-in features to collect potentially PII for users who need it
+- Collect it but with a configuration option to obfuscate / redact the PII, if
+  you’re confident about being able to implement the obfuscation / redaction
+  effectively and with little overhead
+
+PII is data like email addresses, passwords, usernames, custom data in database
+queries, etc. For example, this is a Elasticsearch query with PII redacted that
+would be a span's `db.statement` attribute:
+
+```json
+{ "query": { "match": { "username": "REDACTED" } } }
+```
```

Comment:
Just ran across this PR and curious about this comment.  Is the idea that we don't want instrumentation to obfuscate PII, but instead want it done in otelcol?

Reply:
@cheempz not necessarily. Lots of people don't use the collector, or they have requirements that nothing sensitive ever leaves a network boundary, even if those boundaries are still internal.

Reply:
OK yes that was my understanding too, thanks for clarifying.

Reply:
@svrnm I think it's common to obfuscate the PII in the instrumentation so it's never sent over the network. It therefore might be worth showing an example of obfuscating the PII in this instrumentation guide, instead of showing an example with the collector. What do you think?

Reply:
I would also prefer showing examples of treating PII in the instrumentation, but the problem we have here is not the content but the presentation: We support 11 languages, so you would need to provide 11 examples (in tabs) eventually. The right place to have that is the language-specific doc sections and link out to that. The collector is often a good choice to provide some "middle-ground"

So here's a proposal (please discuss) what we can do instead:
* Provide a list of example redactions & obfuscations in a table, e.g. IP addresses (192.168.0.17 -> 192.168.x.x, or "username" to "REDACTED", or credit card number "4860926956060625" to "Visa"). Again this should be noted as "suggestions" and that people need to be cautious around local laws and making sure that what they do is "safe" (what ever that means in that context)
* Verbally mention that you can do this either in the language specific instrumentation (link to the concept page for instrumentation for now, we **later** can add that to the language specific docs and link to them) and the collector (link to the transformation page)
* Probably it is also worth mentioning that there is some mechanism in Automatic Instrumentation to help you with that (if I remember correctly there is some SQL parameter redaction capabilities in some languages)